src/Symfony/Bundle/DependencyInjection/AwsPackagesProvider.php

```diff
@@ -9,6 +9,14 @@ class AwsPackagesProvider
     public static function getAllServices(): array
     {
         return [
+            'cloudformation' => [
```

Comment:
- cloudFormation?
- cloud_formation?

Reply:
`cloud-formation` ?
or `CloudFormation` like the official bundle https://github.com/aws/aws-sdk-php-symfony

Reply:
This goes into the yaml config. That is why Im asking. Having dash in yaml isnt really nice, is it?

Reply:
We should use snake case to be consistent with other config: 
https://symfony.com/doc/current/reference/configuration/framework.html

Reply:
don't Symfony use snakeCase to separate words? like `max_redirects` but use all lowercase for the same word? like `passphrase`?
just asking.

Reply:
Yes. Symfony use snake_case to separate words. I think `Cloud Formation` is two words.